docker/server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
+
+RUN true \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		git vim-tiny \
+	&& apt-get clean \
+	&& true
+
+RUN true \
+	&& git clone https://github.com/kiwicom/iwant-bot.git /iwant-bot-cloned \
+	&& ln -s /iwant-bot-cloned /iwant-bot \
+	&& (cd iwant-bot && git checkout docker-support) \
+	&& pip install -r iwant-bot/requirements.txt \
+	&& true
+
+CMD [ "python", "/iwant-bot/src/iwant_bot/start.py"]
```

Comment:
Also when we have python package we should start it like this`python -m iwant_bot`